run-tests.php

```diff
@@ -1265,23 +1265,19 @@ function run_test($php, $file, $env)
 
 	$fp = fopen($file, "rb") or error("Cannot open test file: $file");
 
-	$borked = false;
-	$bork_info = '';
+	$bork_info = null;
 
 	if (!feof($fp)) {
 		$line = fgets($fp);
 
 		if ($line === false) {
 			$bork_info = "cannot read test";
-			$borked = true;
 		}
 	} else {
 		$bork_info = "empty test [$file]";
-		$borked = true;
 	}
-	if (!$borked && strncmp('--TEST--', $line, 8)) {
+	if (is_null($bork_info) && strncmp('--TEST--', $line, 8)) {
```

Comment:
we encourage always using `$bork_info === null` rather than `is_null($bork_info)` since the latter one is a function call. isn't it?

Reply:
@jhdxr Function calls are slower than comparison, or isn’t something measurable?

Reply:
@carusogabriel so in fact I'm suggesting using `$bork_info === null` to replace `is_null($bork_info)` here and below.

Reply:
function calls are cheap, but in case you run a lot of them its nevetheless worth saving them.

Reply:
@jhdxr @staabm Thanks, TIL. Gonna use `$bork_info === null`

Reply:
In this case `is_null()` will not actually result in a function call (it compiles to a TYPE_CHECK instruction). I wouldn't be able to guess at which is faster without measuring.

In any case, I think that in modern PHP using `=== null` over `is_null` is a de facto standard, so I'd prefer it as well, independently of micro-optimization...